tests/perf/test_sheriffing_criteria/test_criteria_tracker.py

```diff
@@ -220,17 +224,19 @@ def test_record_computer_can_tell_unallowed_data(criteria_record):
 
 @pytest.mark.freeze_time(CASSETTES_RECORDING_DATE)  # disable tick
 @pytest.mark.parametrize('exception', [NoFiledBugs(), Exception()])
-def test_record_computer_still_updates_if_one_of_the_formulas_fails(exception):
+def test_record_computer_still_updates_if_one_of_the_formulas_fails(exception, db):
```

Comment:
do you use `db` param anywhere?

Reply:
This is part of `pytest-django`'s [database fixtures](https://pytest-django.readthedocs.io/en/latest/helpers.html#pytest-mark-django-db-request-database-access). This tells pytest the function will perform database related operations.